static/app/views/onboarding/targetedOnboarding/components/stepper.tsx

```diff
@@ -26,6 +28,13 @@ const StepperTransitionIndicator = styled(motion.span)`
   background-color: ${p => p.theme.progressBar};
   position: absolute;
 `;
+StepperTransitionIndicator.defaultProps = {
+  layout: true,
+  transition: testableTransition({
+    type: 'tween',
+    duration: 1,
```

Comment:
nit and IMO but this probably should be faster, I would suggest 300ms. I know you want it to kinda match the new page showing up, but it will "feel" better if this is faster IMO.

Reply:
sure we can use 300ms.